sunpy/map/mapbase.py

```diff
@@ -1891,6 +1893,76 @@ def draw_limb(self, axes=None, **kwargs):
 
         return [circ]
 
+    @u.quantity_input
+    def draw_quadrangle(self, bottom_left, *, width: u.deg = None, height: u.deg = None,
```

Comment:
A larger question here: Do we want a new method or do we want to fold this into the existing `draw_rectangle` if we do want a new method do we want to deprecate the old `draw_rectangle`?

Reply:
I list some of the possibilities in the originating issue (#4657), so I suggest you discuss there instead of this PR.